common/flatpak-dir.c

```diff
@@ -9677,7 +9801,19 @@ flatpak_dir_get_locale_subpaths (FlatpakDir *self)
       if (flatpak_dir_is_user (self))
         subpaths = flatpak_get_current_locale_subpaths ();
       else
-        subpaths = g_new0 (char *, 1);
+        {
+          GDBusProxy *proxy = get_accounts_dbus_proxy ();
+          if (proxy)
+            {
+              subpaths = get_locale_subpaths_from_accounts_dbus (proxy);
```

Comment:
Every time the "flatpak_dir_get_locale_subpaths" function is called, this will do N+1 synchronous dbus("N" is "traverse the user language" and  "1" is "get user dbus object path"). 
In my test, during system side installation, the "flatpak_dir_get_locale_subpaths" function was called twice. It means during system side installation, will do 2*(N+1) synchronous dbus.
If company laptop have 150 users, do 302 synchronous dbus need 320-350 ms.
Can these times be ignored compared to file downloads and copies?